### ISSUE TITLE
Request: MissingMoviesSearch

### ISSUE THREAD

**Issue by marc0janssen** [vche/pycliarr#24]
Hi Vivien, 

Can you please implement the following method?

Merci!!

  def MissingMoviesSearch(self) -> json_data:
        """Perform an Missing missing movie search.
        Returns:
            json response
        """
        return self._sendCommand({"name": "MissingMoviesSearch"})

**Comment by vche** [comment]
I didn't add it on purpose because the wanted API just doesn't exist in radarr. As you can see in the UI, sonarr has a "wanted" section, but not radarr.

You can only get the movie list and filter out by status those missing (not downloaded). 
so basiccally you can do something like 

```
missing = [ missing_movie for missing_movie in radarr_cli.get() if missing_movie.downloaded is False]
```

**Comment by vche** [comment]
wait, is there an existing command "MissingMoviesSearch" ?

**Comment by marc0janssen** [comment]
Yes, I implemented in my code... it's a proper command from Radarr.
I thought you might want to implement it too in pycliarr.

**Comment by vche** [comment]
nice. i didn't see it in radarr api. I will implement it, thanks !

**Comment by vche** [comment]
Fixed in v1.0.19
